front/app/containers/Admin/projectFolders/containers/settings/ProjectFolderForm/index.tsx

```diff
@@ -593,9 +595,12 @@ const ProjectFolderForm = ({ mode, projectFolderId }: Props) => {
           ) : (
             <ProjectFolderCardImageDropzone
               images={folderCardImage && [folderCardImage]}
-              onAddImage={getHandler((cards: UploadFile[]) =>
-                setFolderCardImage(cards[0])
-              )}
+              onAddImage={getHandler((cards: UploadFile[]) => {
+                setFolderCardImage(cards[0]);
+                if (cards[0]?.base64) {
+                  setCroppedFolderCardBase64(cards[0].base64);
```

Comment:
I don't fully understand this, could you explain why you added this?

Reply:
So essentially if the Admin uploads an image but doesn't move it around in the Image cropper tool, we never set the `CroppedFolderCardBase64`. In order to properly upload/save the image, the CroppedFolderCardBase64 _needs_ to be set. Prior to my changes, the only way to set the CroppedFolderCardBase64 was to move the image around in the cropper tool a bit after uploading, and only then clicking save. If the admin did that, the image was properly saved. If they didn't though and just uploaded an image and clicked save without interacting with the image cropper, the CroppedFolderCardBase64 was never set.

Here I'm just explicitly setting an initial value for the CroppedFolderCardBase64 as soon as the admin uploads an image (a default crop). :)

Reply:
@amanda-anderson Perhaps it could be useful to add a short comment next to the code, so that everyone understands this (and doesn't remove this code, thinking that it's not doing anything meaningful)? Or do you think this is clear to experienced frontend devs?

Reply:
@amanda-anderson Is this the same issue? https://www.notion.so/citizenlab/Image-resizing-is-confusing-6faf6009da634eb4a815c00dfa76ba50?pvs=4